config.py

```diff
@@ -6,5 +6,5 @@
 SECRET_KEY = 'development key'
 USERNAME = 'admin'
 PASSWORD = 'default'
-AUTH_TOKEN = ''
```

Comment:
This is an installation-specific configuration and probably shouldn't have been included in the commit.